.travis.yml

```diff
@@ -47,12 +47,12 @@ before_install:
 - nvm use $TRAVIS_NODE_VERSION
 
 before_script:
+- npm uninstall -D husky lint-staged
```

Comment:
makes sense removing them upfront, but in practice Travis runs `npm install` as part of the `install` step (which we don't override), so it would make sense to move this command even higher, as the last one in `before_install`

Reply:
Hm, it doesn't seem to be working when I move this to the `before_install`:

The `uninstall` command runs: https://travis-ci.org/jaegertracing/jaeger-client-node/jobs/347115286#L628

But, husky is still installed: https://travis-ci.org/jaegertracing/jaeger-client-node/jobs/347115286#L1186

I'm going to move it back to the first step in the `before_script` so the CI builds pass for the older versions of node and this PR can move forward.